src/utils/api/getBlockchainStatus.ts

```diff
@@ -0,0 +1,19 @@
+//
+// API call helper for getBlockchainStatus, not meant to be called directly (meant to be used inside the APIProvider)
+//
+
+import { IGetBlockchainStatusResult } from "types/NXTAPI";
+import { API } from "./api";
+
+async function getBlockchainStatus(): Promise<false | IGetBlockchainStatusResult> {
+  let result;
+  try {
+    result = await API(`requestType=getBlockchainStatus`, "GET");
```

Comment:
the suggestion for changing API to accept an object with properties (including requestType as the specific strings allowed) is probably best before too long.